core/unit_test/TestCuda_Category.hpp

```diff
@@ -42,13 +42,14 @@
 //@HEADER
 */
 
-#include <cuda/TestCuda_Category.hpp>
-#include <TestSharedAlloc.hpp>
+#ifndef KOKKOS_TEST_CUDA_HPP
+#define KOKKOS_TEST_CUDA_HPP
 
-namespace Test {
+#include <gtest/gtest.h>
 
-TEST(TEST_CATEGORY, impl_shared_alloc) {
-  test_shared_alloc<Kokkos::CudaSpace, Kokkos::DefaultHostExecutionSpace>();
-}
+#define TEST_CATEGORY cuda
+#define TEST_CATEGORY_NUMBER 5
```

Comment:
These numbers are used in `core/unit_test/TestSharedAlloc.hpp`. I am happy to replace it with a better workaround.